modules/post-installation-adding-nutanix-failure-domains-compute-machines.adoc

```diff
@@ -0,0 +1,120 @@
+
+.Prerequisites
+
+* You have configured the failure domains in the cluster's Infrastructure custom resource (CR).
+
+.Procedure
+
+. Run the following command to view the cluster's Infrastructure CR.
++
+[source,terminal]
+----
+$ oc describe infrastructures.config.openshift.io cluster
+----
+. For each failure domain (`platformSpec.nutanix.failureDomains`), note the cluster's UUID, name, and subnet object UUID. These values are required to add a failure domain to a compute machine set.
+. List the compute machine sets in your cluster by running the following command:
++
+[source,terminal]
+----
+$ oc get machinesets -n openshift-machine-api
+----
+. Edit the first compute machine set by running the following command:
++
+[source,terminal]
+----
+$ oc edit machineset <machineset_name> -n openshift-machine-api
+----
+. Configure the compute machine set to use the the first failure domain by adding the following to the `spec.template.spec.providerSpec.value` stanza:
++
+[NOTE]
+====
+Be sure that the values you specify for the `cluster` and `subnets` fields match the values that were configured in the `failureDomains` stanza in the cluster's Infrastructure CR.
+====
++
+.Example compute machine set with Nutanix failure domains
+[source,yaml]
+----
+apiVersion: machine.openshift.io/v1
+kind: MachineSet
+metadata:
+  creationTimestamp: null
+  labels:
+    machine.openshift.io/cluster-api-cluster: <cluster_name>
+  name: <machineset_name>
+  namespace: openshift-machine-api
+spec:
+  replicas: 2
+# ...
+  template:
+    spec:
+# ...
+      providerSpec:
+        value:
+          apiVersion: machine.openshift.io/v1
+          failureDomain:
+            name: <failure_domain_name_1>
+          cluster:
+            type: uuid
+            uuid: <prism_element_uuid_1>
+          subnets:
+          - type: uuid
+            uuid: <prism_element_network_uuid_1>
+# ...
+----
+. Note the value of `spec.replicas`, as you need it when scaling the machine set to apply the changes.
+. Save your changes.
+. List the machines that are managed by the updated compute machine set by running the following command:
++
+[source,terminal]
+----
+$ oc get -n openshift-machine-api machines -l machine.openshift.io/cluster-api-machineset=<machine_set_name>
+----
+. For each machine that is managed by the updated compute machine set, set the `delete` annotation by running the following command:
++
+[source,terminal]
+----
+$ oc annotate machine/<machine_name_original_1> \
+  -n openshift-machine-api \
+  machine.openshift.io/delete-machine="true"
+----
+. Scale the compute machine set to twice the number of replicas by running the following command:
++
+[source,terminal]
+----
+$ oc scale --replicas=<twice_the_number_of_replicas> \// <1>
+  machineset <machine_set_name> \
+  -n openshift-machine-api
+----
+<1> For example, if the original number of replicas in the compute machine set was `2`, scale the replicas to `4`.
```

Comment:
```suggestion
<1> For example, if the original number of replicas in the compute machine set is `2`, scale the replicas to `4`.
```

Reply:
Do we need to explain why we need to scale to double, or would a typical user know?

Reply:
No I do not believe it is necessary to explain why. If we get feedback after this docs go live that there is confusion, then we can update accordingly.